src/components/PlayerCardContainer/PlayerCardForm.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import './PlayerCardForm.css'
+import CustomButton from "./CustomButton";
+
+function PlayerCardForm(props) {
```

Comment:
This is named Form, but only does display. The other one is named PlayerCard, but has an input form.

Reply:
I refactored PlayerCardForm to PlayerCardDisplay and PlayerCard to PlayerCardForm, thoughts?

Reply:
Makes sense to me :)